flake.nix

```diff
@@ -50,6 +50,8 @@
       dell-xps-15-7590-nvidia = import ./dell/xps/15-7590/nvidia;
       dell-xps-15-9500 = import ./dell/xps/15-9500;
       dell-xps-15-9500-nvidia = import ./dell/xps/15-9500/nvidia;
+      dell-xps-15-9510 = import ./dell/xps/15-9510;
```

Comment:
Please also add this to the README table where our CI picks it up for testing.

Reply:
Done! Thanks for letting me know @Mic92 \m/